pkg/operator/observe_config.go

```diff
@@ -157,6 +166,42 @@ func observeInternalRegistryHostname(listers Listers, observedConfig map[string]
 	return observedConfig, nil
 }
 
+func observeRoutingSubdomain(listers Listers, observedConfig map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
@sanchezl figure out what state this config observation is in to finish this review